zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/cache/ProcessCacheMetrics.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.exporter.cache;
+
+import com.github.benmanes.caffeine.cache.RemovalCause;
+import com.github.benmanes.caffeine.cache.stats.CacheStats;
+import com.github.benmanes.caffeine.cache.stats.StatsCounter;
+import io.micrometer.core.instrument.Counter;
+import io.micrometer.core.instrument.MeterRegistry;
+import io.micrometer.core.instrument.Timer;
+import java.util.concurrent.TimeUnit;
+
+public class ProcessCacheMetrics implements StatsCounter {
+
+  private static final String NAMESPACE = "zeebe.camunda.exporter.processcache";
+
+  private final Counter hitCount;
+  private final Counter missCount;
+  private final Timer loadSuccessDuration;
+  private final Timer loadFailureDuration;
+  private final Counter evictionCount;
+
+  public ProcessCacheMetrics(final MeterRegistry meterRegistry) {
+    hitCount =
+        Counter.builder(meterName("hits"))
+            .description("Number of cache hits")
+            .register(meterRegistry);
+
+    missCount =
+        Counter.builder(meterName("misses"))
+            .description("Number of cache misses")
+            .register(meterRegistry);
+
+    evictionCount =
+        Counter.builder(meterName("evictions"))
+            .description("Number of cache evictions")
+            .register(meterRegistry);
+
```

Comment:
Can we also handle via a label, type

Reply:
It looked like we had to define tags directly with values. So I first thought this might not feasible, BUT actually it is just to define the type (it seems to be).

See https://stackoverflow.com/a/49171202/2165134

Reply:
What I think we can do is: 

```

access = Counter.builder(meterName("access"))
            .description("Number of cache access by type")
            .tag("type", "")
            .register(meterRegistry);
```

And then later:

```
  @Override
  public void recordHits(final int count) {
    Metrics.counter(meterName("access"), "type", "hit").increment(count);
  }
```

Reply:
I guess one issue with this is again using static registry 🤔 instead of the actual live object?

Reply:
Thanks to @lenaschoenburg I realized we can use the actual registry we have at the hand to do the same.